pkg/docker/docker.go

```diff
@@ -95,11 +121,11 @@ func (bm *BasicManager) ContainerAbsent(ctx context.Context, containerName strin
 	if running {
 		fmt.Printf("Stopping container '%s'\n", containerName)
 
-		if err := bm.cli.ContainerStop(ctx, containerName, nil); err != nil {
+		if err := bm.cli.ContainerStop(ctx, bm.prefixedName(containerName), nil); err != nil {
```

Comment:
Since `bm.prefixedName(containerName)` is re-used everywhere, you can probably just move it up:

```go
pn := bm.prefixedName(containerName)
```

Reply:
fixed